utils/core-contracts/contracts/utils/RevertUtil.sol

```diff
@@ -0,0 +1,15 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.11 <0.9.0;
+
+/*
+		Utilities related to reverts and error handling
+*/
+
+library RevertUtil {
```

Comment:
Did this get added by mistake? Harmless, but seems out of scope/unused

Reply:
ah whoops. I think I checked it in because I thought I forgot to include the file. probably goes with a different PR :/